website/content/docs/examples/bootstrap-k0s.md

```diff
@@ -71,6 +77,23 @@ bctl apply -f k0s-in-aws-with-tf.yaml
 bctl update -f k0s-in-aws-with-tf.yaml
 ```
 
+5. View the status of the cluster's Kubernetes pods:
```

Comment:
We should actually make this match the other examples that watch for the pods to be read so the docs are consistent


It will take a few moments before the pods are ready. You can monitor the progress with:

```shell
watch -n 1 kubectl get pods -n shipa-system
```